blueman/main/PluginManager.py

```diff
@@ -23,21 +23,18 @@ class LoadException(Exception):
     pass
 
 
-T = TypeVar('T', AppletPlugin, ManagerPlugin)
-
-
-class PluginManager(GObject.GObject, Generic[T]):
+class PluginManager(GObject.GObject):
     __gsignals__: GSignals = {
         'plugin-loaded': (GObject.SignalFlags.NO_HOOKS, None, (GObject.TYPE_STRING,)),
         'plugin-unloaded': (GObject.SignalFlags.NO_HOOKS, None, (GObject.TYPE_STRING,)),
     }
 
     def __init__(self, plugin_class, module_path, parent):
         super().__init__()
-        self.__plugins: Dict[str, T] = {}
-        self.__classes: Dict[str, Type[T]] = {}
-        self.__deps: Dict[str, str] = {}
-        self.__cfls: Dict[str, str] = {}
+        self.__plugins: Dict[str, Union[AppletPlugin, ManagerPlugin]] = {}
+        self.__classes: Dict[str, Type[Union[AppletPlugin, ManagerPlugin]]] = {}
+        self.__deps: Dict[str, List[str]] = {}
```

Comment:
Fix also in #1122.

Reply:
Ooh, I forgot to take this out. Will do before merging!

Reply:
Hidden fixes welcome. 😉